### PR TITLE
Update wise.md

### PR DESCRIPTION
**Pull request by Vad1mo**
Add SCA related section

### CODE REVIEW

File: docs/integrations/builtin/credentials/wise.md [hunk 1]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA protected endpoint, a 403 Forbidden HTTP status code is returned.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:44]:
```suggestion
When making a request to an SCA protected endpoint, Wise returns a 403 Forbidden HTTP status code.
```

File: docs/integrations/builtin/credentials/wise.md [hunk 2]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:45]: Heya! Not 100% clear what is being referred to here. If possible, could you match to the operation names in n8n? So for example, if SCA is required for Transfer > Create operations, or for all Transfer operations?

File: docs/integrations/builtin/credentials/wise.md [hunk 3]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA protected endpoint, Wise returns a 403 Forbidden HTTP status code.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:44]:
```suggestion
When making a request to an SCA (strong customer authentication) protected endpoint, Wise returns a 403 Forbidden HTTP status code.
```

File: docs/integrations/builtin/credentials/wise.md [hunk 4]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA (strong customer authentication) protected endpoint, Wise returns a 403 Forbidden HTTP status code.
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
+
+If SCA is required an error similar ot below returnd from Wise:
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:47]:
```suggestion
If SCA is required, Wise returns an error similar to below : 
```

File: docs/integrations/builtin/credentials/wise.md [hunk 5]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+
+When making a request to an SCA (strong customer authentication) protected endpoint, Wise returns a 403 Forbidden HTTP status code.
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
+
+If SCA is required an error similar ot below returnd from Wise: 
+
+> This request requires Strong Customer Authentication (SCA). Please add a key pair to your account and n8n credentials. See https://api-docs.transferwise.com/#strong-customer-authentication-personal-token
+
+
+To enable signing a token we need to create and add a public and private key  and add the public key to wise. In user [profile settings](https://wise.com/settings/public-keys).
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:52]:
```suggestion
To enable signing a token you need to create and add a public and private key. Add the public key to Wise in user [profile settings](https://wise.com/settings/public-keys){:target=_blank .external-link}.
```

File: docs/integrations/builtin/credentials/wise.md [hunk 6]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA (strong customer authentication) protected endpoint, Wise returns a 403 Forbidden HTTP status code.
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
+
+If SCA is required, Wise returns an error similar to below :
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:47]:
```suggestion
If SCA is required, Wise returns an error similar to below: 
```

File: docs/integrations/builtin/credentials/wise.md [hunk 7]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+SCA is required on Get Statements, and Funds transfer request. It is disabled when using test endpoints.
+
+If SCA is required, Wise returns an error similar to below: 
+
+> This request requires Strong Customer Authentication (SCA). Please add a key pair to your account and n8n credentials. See https://api-docs.transferwise.com/#strong-customer-authentication-personal-token
+
+
+To enable signing a token you need to create and add a public and private key. Add the public key to Wise in user [profile settings](https://wise.com/settings/public-keys){:target=_blank .external-link}.
+
+As a first step we need to generate an RSA key pair:
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:54]:
```suggestion
To generate an RSA key pair:
```

File: docs/integrations/builtin/credentials/wise.md [hunk 8]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+
+To enable signing a token you need to create and add a public and private key. Add the public key to Wise in user [profile settings](https://wise.com/settings/public-keys){:target=_blank .external-link}.
+
+To generate an RSA key pair:
+
+```sh
+$ openssl genrsa -out private.pem 2048 
+$ openssl rsa -pubout -in private.pem -out public.pem
+```
+With the generated keys, we now need to add them to n8n and Wise.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:60]:
```suggestion
With the generated keys, add them to n8n and Wise:
```

File: docs/integrations/builtin/credentials/wise.md [hunk 9]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+
+To generate an RSA key pair:
+
+```sh
+$ openssl genrsa -out private.pem 2048 
+$ openssl rsa -pubout -in private.pem -out public.pem
+```
+With the generated keys, add them to n8n and Wise:
+
+- Add the content of the public key `public.pem` to your Wise user [profile settings](https://wise.com/settings/public-keys).
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:62]:
```suggestion
- Add the content of the public key `public.pem` to your Wise user [profile settings](https://wise.com/settings/public-keys){target=_blank .external-link}.
```

File: docs/integrations/builtin/credentials/wise.md (hunk 2 above)
Comment (Vad1mo) [reply docs/integrations/builtin/credentials/wise.md:45]: Its not clear where SCA is needed. So and to my knowledge  it is `Get Statements` and `Funds`

File: docs/integrations/builtin/credentials/wise.md [hunk 10]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
...
+To generate an RSA key pair:
+
+```sh
+$ openssl genrsa -out private.pem 2048 
+$ openssl rsa -pubout -in private.pem -out public.pem
+```
+With the generated keys, add them to n8n and Wise:
+
+- Add the content of the public key `public.pem` to your Wise user [profile settings](https://wise.com/settings/public-keys){:target=_blank .external-link}.
+- Add the content of the private key `private.pem` to your n8n Wise Credential under `Private Key (Optional)`
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:63]:
```suggestion
- Add the content of the private key `private.pem` to your n8n Wise Credential under **Private Key (Optional)**.
```

File: docs/integrations/builtin/credentials/wise.md (hunk 2 above)
Comment (StarfallProjects) [reply docs/integrations/builtin/credentials/wise.md:45]:
```suggestion
SCA is required on some operations. Refer to [Wise documentation | Strong Customer Authentication & 2FA](https://docs.wise.com/api-docs/features/strong-customer-authentication-2fa) for details. It is disabled when using test endpoints.
```

File: docs/integrations/builtin/credentials/wise.md [hunk 11]
```diff
@@ -38,3 +38,29 @@ The following video demonstrates the steps mentioned above.
 <iframe width="840" height="472.5" src="https://www.youtube.com/embed/hys2lDEScUE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
+
+## Personal Token SCA
+
+When making a request to an SCA (strong customer authentication) protected endpoint, Wise returns a 403 Forbidden HTTP status code.
+SCA is required on some operations. Refer to [Wise documentation | Strong Customer Authentication & 2FA](https://docs.wise.com/api-docs/features/strong-customer-authentication-2fa) for details. It is disabled when using test endpoints.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/wise.md:45]:
```suggestion
SCA is required on some operations. Refer to [Wise documentation | Strong Customer Authentication & 2FA](https://docs.wise.com/api-docs/features/strong-customer-authentication-2fa){:target=_blank .external-link} for details. It is disabled when using test endpoints.
```

**Review by StarfallProjects — APPROVED** [review]

**Comment by Vad1mo** [comment]
Awesome thank you

**Pull request merged**